src/ui/components/Dashboard/RecordingItem/RecordingListItem.js

```diff
@@ -86,7 +86,7 @@ function ItemTitle({ data, editing, editingTitle, setEditingTitle }) {
           </div>
         ) : null}
       </div>
-      <div className="page-url">Created {moment(data.date).fromNow()}</div>
```

Comment:
do we need to carry over the styling for `page-url`?

Reply:
Yeah we should be able to just take out the classname here. The styling's being handled here without having to refer to a classname.

Reply:
no i checked with and without and it doesn't alter the styling at all. the only styling attached to that classname is changing the color to blue and underlining the text on hover